lib/blackbeard/math.rb

```diff
@@ -0,0 +1,14 @@
+require 'bigdecimal/math'
+
+module Blackbeard
+  module Math
+    def self.beta(x,y)
+      lx = ::Math.lgamma(x).first
+      ly = ::Math.lgamma(y).first
+      lxplusy = ::Math.lgamma(x + y).first
+
+      precision = 5
+      ::BigMath.exp(lx + ly - lxplusy, precision)
+    end
+  end
```

Comment:
Explanation for why this function exists is [here](https://github.com/goldstar/blackbeard/blob/9af9ff46e841ae82d2819db8a02b30b738a58d93/docs/beta.org)